Extensions/InternetConnectivity.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "The Gem Dev",
+  "description": "",
```

Comment:
Set description to `A condition to check if the device is connected to the internet.`